config/fees.php

```diff
@@ -1,5 +1,12 @@
 <?php
 
 return [
+    //Users Model
+    'user_model' => \App\Models\User::class,
```

Comment:
Should be updated accordingly to https://www.notion.so/tger/Rules-c1a7ceb5485048d0b2b68ed4247f887b

Reply:
@sl0wik Updated in my latest commit.